requirements/requirements.in

```diff
@@ -3,5 +3,7 @@ factory_boy
 faker
 importlib-metadata >= 3.6.0 ; python_version < "3.10"
 numpy
+pip-tools
+pre-commit
```

Comment:
Adding these for now since I'm unsure if they should be used for lint and compile commands.